product_supplierinfo_for_customer/models/product_product.py

```diff
@@ -114,3 +103,30 @@ def price_compute(self, price_type, uom=False, currency=False, company=False):
         return super(ProductProduct, self).price_compute(
             price_type, uom, currency, company
         )
+
+    def _prepare_customerinfo(self, params):
```

Comment:
Why this double method? If you want to inject things on the domain for performing only one search, then put a `_prepare_domain` hook, but this is not adding anything useful to the inheritance (nothing that you can't done on the main method).

Reply:
Done.